app-samples/FaceAPIHeadPoseSample/FaceAPIHeadPoseSample/MainWindow.xaml.cs

```diff
@@ -0,0 +1,695 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.IO.IsolatedStorage;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using Microsoft.Azure.CognitiveServices.Vision.Face;
+using Microsoft.Azure.CognitiveServices.Vision.Face.Models;
+using OpenCvSharp;
+using OpenCvSharp.Extensions;
+
+namespace FaceAPIHeadPoseSample
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : System.Windows.Window, INotifyPropertyChanged
+    {
+        #region Fields
+
+        private static FrameGrabber<LiveCameraResult> _grabber = null;
+        private static readonly ImageEncodingParam[] s_jpegParams = {
+            new ImageEncodingParam(ImwriteFlags.JpegQuality, 60)
+        };
+
+        private readonly string _isolatedStorageSubscriptionKeyFileName = "Subscription.txt";
+        private readonly string _isolatedStorageSubscriptionEndpointFileName = "SubscriptionEndpoint.txt";
+
+        private readonly double _HeadPitchMaxThreshold = 30;
```

Comment:
`_headPitchMaxThreshold`, same for others.

Reply:
Done.